integration-tests/integration/rewrite.spec.js

```diff
@@ -0,0 +1,43 @@
+const HomePage = require('../pages/homePage')
+const OffenderSearchPage = require('../pages/offenderSearchPage')
+const KeyworkerProfilePage = require('../pages/keyworkerProfilePage')
+const Utils = require('../support/utils')
+const KeyworkerAllocationsResponse = require('../responses/keyworkerAllocationsResponse')
```

Comment:
We normally include the responses in the spec file rather than put them in a different file. The reason for this - I believe - is that the response determines what is shown and hence what is asserted later on in the file, so it is directly connected.

Reply:
So the reason I've done this is because the groovy tests basically had a bunch of data which they then used in multiple different tests, and so we've ended up with the same issue here. i.e. the same data is used in different specs. I more than happy to change it up so that we have the data in the specs themselves (as I agree that they are directly connected) but it will mean that we duplicate the data. If you're happy with that then I'll go ahead